dummyserver/server.py

```diff
@@ -40,6 +40,20 @@
 DEFAULT_CA_KEY = os.path.join(CERTS_PATH, "cacert.key")
 
 
+def _sane_ipv6_dns():
+    """ Returns True if the system resolves localhost to an IPv6 address by default. """
+    sane_ipv6_dns = False
+    try:
+        for res in socket.getaddrinfo(host, None, socket.AF_UNSPEC):
```

Comment:
Don't we want to pass `host` to this function? And then replace "localhost" with "host" in the docstring?

Reply:
You are right :) done.